ReactNativeTemplate/android/app/build.gradle

```diff
@@ -109,6 +109,11 @@ android {
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
         }
+        debug {
+            defaultConfig {
+                targetSdkVersion 22
```

Comment:
I'm okay with using RN target sdk version. Our SDK will work with 22, we don't know if theirs will work with 25. So it makes sense to go with the one we know will work for both.